utils/unittest/version_beacon.go

```diff
@@ -55,7 +55,7 @@ func AddVersionBeacon(t *testing.T, beacon *flow.VersionBeacon, state protocol.F
 
 func addToState(t *testing.T, state protocol.FollowerState, block *flow.Block, finalize bool) {
 
```

Comment:
Big fan of concise code (avoiding blank lines unless they are for separating the code into chunks that do different things)
```suggestion
```